CHANGELOG.md

```diff
@@ -1,3 +1,25 @@
+# OP-TEE - version 3.7.0 (2019-10-11)
+
+- Link to the GitHub [release page][github_release_3_7_0] (optee_os)
```

Comment:
Why only link to the release page of optee_os and not optee_client etc.?

Reply:
I can do it, I'll update the script.

Reply:
Done.